test_communication/test/test_publisher_subscriber.cpp

```diff
@@ -33,8 +33,8 @@ void publish(
   auto publisher = node->create_publisher<T>(
     std::string("test_message_") + message_type, custom_qos_profile);
 
-  rclcpp::WallRate time_between_cycles(1);
-  rclcpp::WallRate time_between_messages(10);
+  rclcpp::WallRate time_between_cycles(10);
```

Comment:
While we're looking at this test, this variable has a misleading name. `time` implies it is some number of seconds, when actually it represents a rate.